INSTALL.md

```diff
@@ -1,5 +1,84 @@
-Building Dogecoin
-================
+# Building Dogecoin Core
 
-See doc/build-*.md for instructions on building the various
-elements of the Dogecoin Core reference implementation of Dogecoin.
+Development is ongoing, and the development team, as well as other volunteers,
+can freely work in their own trees and submit pull requests when features or
+bug fixes are ready.
+
+### Wow plz make dogecoind/dogecoin-cli/dogecoin-qt
+
+  The following are developer notes on how to build Dogecoin on your native platform. They are not complete guides, but include notes on the necessary libraries, compile flags, etc.
+
+  - [OSX Build Notes](doc/build-osx.md)
+  - [Unix Build Notes](doc/build-unix.md)
+  - [Windows Build Notes](doc/build-windows.md)
+
+### Such ports
+
+- RPC 22555
+- P2P 22556
+
+#### Version strategy
```

Comment:
Is installation guide where people will look for development steps? I'm almost tempted to make a specific development guide for this is to move out of the main readme?

Reply:
Yeah, definitively this INSTALL.md is still messy. It's more Work in progress, I moved all technical information for "developers" in the install section, to keep all infos, and wanted to rewrite the `INSTALL.md` after that.

This PR, at the beginning, was more focused on the README to make it accessible for every type of shibes.